presto-verifier/src/main/java/com/facebook/presto/verifier/framework/Column.java

```diff
@@ -86,10 +77,10 @@ public Type getType()
         return type;
     }
 
-    public static Column fromResultSet(ResultSet resultSet)
+    public static Column fromResultSet(TypeManager typeManager, ResultSet resultSet)
```

Comment:
Since `FunctionManager` now requires `TransactionManager` to construct. We need a more formal construction of `TypeManager` and `FunctionManager` through injection.